code/style.css

```diff
@@ -41,7 +41,7 @@ select {
 }
```

Comment:
Hey Katie, I am not familiar with the @-thing. What does it do? 😅

Reply:
Hey Jessi, I don't know neither.. 😆  I think it's something GitHub does. I double checked my code and I haven't those @thing 😬